frontend/src/ui/wallet/transfer/transfer-settings.js

```diff
@@ -139,6 +139,7 @@ export default class TransferSettings {
             this.createTrustline = false
         }
         this.asset[1] = this.asset[0]
+        this.amount[1] = this.amount[0]
```

Comment:
Could you please explain this?

Reply:
Amount need reset like asset, coz if form filled and just swaped tabs of transfer form (ex: convert -> claimable -> convert): selected assets is reseted, but amount - not.

1 XLM ~ 0.8 USDC and after swapping of tabs will be shown on form 1 XLM ~ 0.8 XML